generate-snapshot.test.js

```diff
@@ -6,14 +6,47 @@ import { generateSnapshot } from "./generate-snapshot.js";
 vi.mock("./user-processing.js");
 
 describe("generateSnapshot", () => {
-    it("prints hello world", () => {
+    it("creates an empty snapshot when given no users", () => {
         console.log = vi.fn();
 
-        when(getUserValues).calledWith("foo").thenReturn("bar");
-
-        generateSnapshot("foo");
+        generateSnapshot([]);
 
         expect(console.log).toHaveBeenCalledTimes(1);
-        expect(console.log).toHaveBeenCalledWith("bar");
+        expect(console.log).toHaveBeenNthCalledWith(
+            1,
+            "EXTERNAL_PERSON_KEY|USER_ID|FIRSTNAME|LASTNAME|STUDENT_ID",
+        );
+    });
+
+    it("creates a snapshot with a single user", () => {
+        console.log = vi.fn();
```

Comment:
Since we're mocking out `console.log` in the same way for each test, I wonder if we could do it once for all tests, rather than in each test?